.eslintrc

```diff
@@ -0,0 +1,32 @@
+{
+  "extends": ["eslint:recommended", "prettier"],
+  "env": { "es6": true, "browser": true, "node": true, "jest": true },
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "overrides": [
```

Comment:
there should be no override if we only use 1 parser. So remove this override and make ESLint only use `@typescript-eslint/parser`